src/devices/auriol_4ld5661.c

```diff
@@ -41,9 +45,10 @@ static int auriol_4ld5661_decode(r_device *decoder, bitbuffer_t *bitbuffer)
 
         uint8_t *b  = bitbuffer->bb[i];
         int id      = b[0];
-        int batt_ok = b[1] >> 7;
+        int battery  = b[1] & 0x80;
```

Comment:
Don't change this line

Reply:
Would be necessary to support the battery bit of both auriol and sempre

Reply:
`>> 7` is the same bit as `0x80` just a different output place.